feat: SelectEmotion 컴포넌트 마크업

gmlyb1 opened the pull request:
### 제목 : [feat: SelectEmotion 컴포넌트 마크업]

### PR 유형

- [x] 새로운 기능 추가
- [ ] 버그 수정
- [x] CSS 등 사용자 UI 디자인 변경
- [ ] 코드에 영향을 주지 않는 변경사항(오타 수정, 탭 사이즈 변경, 변수명 변경)
- [ ] 코드 리팩토링
- [ ] 주석 추가 및 수정
- [ ] 문서 수정
- [ ] 테스트 추가, 테스트 리팩토링
- [ ] 빌드 부분 혹은 패키지 매니저 수정
- [ ] 파일 혹은 폴더명 수정
- [ ] 파일 혹은 폴더 삭제

### PR 체크리스트

<!-- PR이 다음 요구 사항을 충족하는지 확인하세요. -->

- [ ] 커밋 메시지 컨벤션에 맞게 작성했습니다.
- [ ] 변경 사항에 대한 테스트를 했습니다.(버그 수정/기능에 대한 테스트).

### 🔎 작업 내용

<!-- 어떤 작업을 하셨는지 상세하게 작성해주세요-->

- SelectEmotion 마크업, 스타일링
- 최초 default gray 처리
- 클릭시 original class 분기 처리
  <br />

### 🔧 앞으로의 과제

- 내일 할 일을 적어주세요
  <br />

### 이슈

<!-- 이슈 키워드와 함께 #을 입력한 후 이슈 번호를 선택해주세요. -->
<!-- 에시 : resolves #1 -->

resolves #40

gmlyb1 committed:
SelectEmotion
- 마크업 및 스타일링
- default시 gray 처리
- 클릭시 원래 color active

gmlyb1 committed:
Merge branch 'develop' into feature/selectEmotion

devHyeon99 commented on src/components/atomic/SelectEmotion.jsx:
```diff
@@ -0,0 +1,56 @@
+import PropTypes from 'prop-types';
+import { useState } from 'react';
+import { HappyMong } from '@/assets/icons/mood';
```
import moodIcons from '@/assets/icons/mood/moodIcons';

여기 엔트리에서 모듈 가져와서 사용하도록 변경해주세요~

devHyeon99 commented on src/components/atomic/SelectEmotion.jsx:
```diff
@@ -0,0 +1,56 @@
+import { SadMong } from '@/assets/icons/mood';
+
+const emotions = [
+  { name: 'happy', src: HappyMong, alt: 'Happy icon', color: '#FFD700' },
```
color값은 쓰이는곳이 없는거 같은데 빼도 될거같아용

devHyeon99 commented on src/components/atomic/SelectEmotion.jsx:
```diff
@@ -0,0 +1,56 @@
+
+  return (
+    <div className="w-[356px] h-[108px] px-5 py-2.5 bg-white rounded-[15px] shadow flex-col justify-center items-center gap-[15px] inline-flex">
+      <div className="text-[#555555] text-base font-semibold font-['Pretendard']">
```
font-['Pretendard']는 선언하실 필요 없어요! 글로벌 적용이 되어 있습니다.

text컬러도 지금 #555로 주셨는데 tailwind.config.js에 있는 gray값의 클래스 이름을 사용해주세요

devHyeon99 commented on src/components/atomic/SelectEmotion.jsx:
```diff
@@ -0,0 +1,56 @@
+  };
+
+  return (
+    <div className="w-[356px] h-[108px] px-5 py-2.5 bg-white rounded-[15px] shadow flex-col justify-center items-center gap-[15px] inline-flex">
```
shadow -> shadow-light 로 사용하셔야 합니다!

devHyeon99 commented on src/components/atomic/SelectEmotion.jsx:
```diff
@@ -0,0 +1,56 @@
+      <div className="text-[#555555] text-base font-semibold font-['Pretendard']">
+        {title}
+      </div>
+      <div className="justify-start items-center gap-[17px] inline-flex">
```
inline-flex 말고 전부 동일하게 flex로 하셔도 될거 같습니당

gmlyb1 committed:
SelectEmotion
- icon 엔트리 모듈 수정
- 필요없는 color 삭제
- text color 일부 수정
- shadow -> shadow-light
- inline-flex -> flex

gmlyb1 committed:
Merge remote-tracking branch 'origin/develop' into feature/selectEmotion
